resources/testdata/glyphs2/WghtVar_AxisMappings.glyphs

```diff
@@ -11,6 +11,10 @@ value = (
 {
 Name = Weight;
 Tag = wght;
+},
+{
+Name = "Optical Size";
+Tag = opsz;
```

Comment:
The current font had axis mappings for opsz but didn't declare it. Since the fields are supposed to align I figured I should correct it by declaring opsz. I will rename the file and add a master that doesn't use the default.

Reply:
Fixed